<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="Initial_nofiletext">
      <property name="styleSheet">
       <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Choose a file to Display</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="7" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QWidget" name="top_left_widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QComboBox" name="topLeftView_comboBox">
                    <property name="minimumSize">
                     <size>
                      <width>200</width>
                      <height>0</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>Axial</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Coronal</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Sagittal</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="top_left_minimize_button">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                    </property>
                    <property name="text">
                     <string>ðŸ—•</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="topleft_maximizebutton">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                    </property>
                    <property name="text">
                     <string>ðŸ—–</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="topLeftView_gridLayout"/>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>98</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="topLeftView_pauseButton">
                    <property name="styleSheet">
                     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                    </property>
                    <property name="text">
                     <string>â–º/||</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="topLeftView_stopButton">
                    <property name="styleSheet">
                     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                    </property>
                    <property name="text">
                     <string>â– </string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>118</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="topLeftView_verticalSlider">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="top_right_widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QComboBox" name="topRightView_comboBox">
                   <item>
                    <property name="text">
                     <string>Coronal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Axial</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Sagittal</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="top_right_minimize_button">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>ðŸ—•</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="topright_maximizebutton">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>ðŸ—–</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="topRightView_gridLayout"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>98</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="topRightView_pauseButton">
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>â–º/||</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="topRightView_stopButton">
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>â– </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>108</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="topRightView_verticalSlider">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QWidget" name="bottom_left_widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QComboBox" name="bottomLeftView_comboBox"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottom_left_minimize_button">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>ðŸ—•</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomleft_maximizebutton">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>ðŸ—–</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="bottomLeftView_gridLayout"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>98</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomLeftView_pauseButton">
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>â–º/||</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomLeftView_stopButton">
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>â– </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_9">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>108</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="bottomLeftView_verticalSlider">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="bottom_right_widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QComboBox" name="bottomRightView_comboBox">
                   <property name="currentText">
                    <string>Sagittal</string>
                   </property>
                   <item>
                    <property name="text">
                     <string>Sagittal</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Axial</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Coronal</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottom_right_minimize_button">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>ðŸ—•</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttomright_maximizebutton">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>ðŸ—–</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="bottomRightView_gridLayout"/>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <spacer name="horizontalSpacer_11">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>98</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomRightView_pauseButton">
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>â–º/||</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bottomRightView_stopButton">
                   <property name="styleSheet">
                    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
                   </property>
                   <property name="text">
                    <string>â– </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_12">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>118</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="bottomRightView_verticalSlider">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>298</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>318</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBrowse_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuBrowse_File"/>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
